lib/operations/replset_get_status.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const ExecuteDbAdminCommandOperation = require('./execute_db_admin_command');
+
+class ReplSetGetStatusOperation extends ExecuteDbAdminCommandOperation {
+  constructor(admin, options) {
+    super(admin.s.db, { replSetGetStatus: 1 }, options);
```

Comment:
since we're going to refactor all of these to use a `CommandOperation` anyway, can you please just create these ones in-place in the code, like:
```
const operation = new ExecuteDbAdminCommandOperation(db, { replSetGetStatus: 1 } options);
...
```